addons/website_slides/models/slide_slide.py

```diff
@@ -140,6 +146,8 @@ class Slide(models.Model):
     quiz_third_attempt_reward = fields.Integer("Reward: third attempt", default=5,)
     quiz_fourth_attempt_reward = fields.Integer("Reward: every attempt after the third try", default=2)
     # content
+    can_be_undone = fields.Boolean('Can Be Undone', compute='_compute_member_actions')
+    can_be_skipped = fields.Boolean('Can Be Skipped', compute='_compute_member_actions')
```

Comment:
Still think we should rename those.
They are mostly the contrary of each other but currently it doesn't really feel like it.

"can_self_mark_completed"
"can_self_mark_uncompleted"

With a small help indicating what it implies?

And the compute to "_compute_mark_complete_actions"?